### ISSUE TITLE
Kucoin order limit issue

### ISSUE THREAD

**Issue by MarcelBeining** [ccxt/ccxt#4139]
- OS: Windows 10
- Programming Language version: Python 3.6
- CCXT version: 1.17.480
- Exchange: kucoin 
- Method: fetchClosedOrders

When I give a limit greater than 20 to fetchClosedOrders the length of the returned array is always 12, instead of the limit number.

```
 len(exchange.fetchClosedOrders(symbol='ETH/USDT',limit=15))
Out[50]: 15
len(exchange.fetchClosedOrders(symbol='ETH/USDT',limit=21))
Out[51]: 12
len(exchange.fetchClosedOrders(symbol='ETH/USDT',limit=100))
Out[52]: 12
```

By the way, there is something I do not understand:
According to https://kucoinapidocs.docs.apiary.io/#reference/0/trading/list-dealt-orders(merged)
there are two ways of fetching dealt orders:
- List dealt orders(merged) GET https://api.kucoin.com/v1/order/dealt (which can take a symbol optionally but this has a limit of 20)
- List dealt orders(specific symbol) GET https://api.kucoin.com/v1/deal-orders (which requires a symbol but there seems to be no limit)
However, I guess ccxt only implemented the first one (as there is this weird described problem with limit>20), right? Or is there a way to request the second way?

**Comment by MarcelBeining** [comment]
Problem still persists with newest ccxt version 1.17.529

**Comment by MarcelBeining** [comment]
any news on that?

**Comment by kroitor** [comment]
@MarcelBeining not yet, it's an issue on their side mostly, so there's little we can do on our side about it, but we hope that this issue will be resolved with the launch of KuCoin Platform 2.0 soon. More about it here: https://github.com/ccxt/ccxt/issues/4514

**Comment by kroitor** [comment]
Upgraded to KuCoin Platform 2.0: https://github.com/ccxt/ccxt/issues/4731
